pages/enroll/lti.ejs

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%- include('../partials/head', {pageTitle: 'Courses'}); %>
+
+    <style>
+      .modal-course-title-style {
+        font-size: 125%;
+      }
+    </style>
+  </head>
+  <body>
+    <script>
+      $(function () {
+          $('[data-toggle="popover"]').popover({sanitize: false})
+      });
+    </script>
```

Comment:
This looks like it's unused on this page.
```suggestion
```

Reply:
@trombonekenny saw this comment was marked resolved - is there a reason this needs to be here?